### ISSUE TITLE
Default Workspaces - Add back viewport visibility toggle

### ISSUE THREAD

**Issue by Draise14** [Bforartists/Bforartists#3338]
![image](https://user-images.githubusercontent.com/35357409/205980633-5147ce62-ddc3-4778-b140-b5338e69a89a.png)

In an update, we lost these toggles. It's a regression. Users in Rclick select have asked for this, we did implement this - probably should add it back.

**Comment by ReinerBforartists** [comment]
BFA 3 never had it activated by default. So no regression here. And i am not sure if it is an improvement really. It adds noise.

**Comment by Draise14** [comment]
Bforartists 1 and 2 had this. Weird it got lost. I find this very useful and necessary. The visibility here can be animated and is stored in file. Hide by eye does not.

**Comment by ReinerBforartists** [comment]
Nope, 2 did not have it neither. Might be that your custom settings tricks your memory here. I just checked the old versions :)

As told, it adds lots of visual noise. And i am not sure if it justifies the change. Hiding it from the viewport is already what you need to make an object invisible in the viewport. Deactivate it in the viewport is not needed in the very most cases. And in the rare cases you need it you can simply turn it on.

So i would vote to not do this task here.

**Comment by ReinerBforartists** [comment]
But nobody moaned so far except you ^^

**Comment by Draise14** [comment]
That was another case. 

I rely on these operators often, and rebuild from factory often every point update - and changing more than a dozen toggles in a dozen workspaces is a chore. It breaks workflow to have to change and toggle these over and over just to animate. And not having them default is also broken UX when it comes to visibility animation typical of mograph where you animate both render and viewport visibility. The visual noise is not disturbing. A few users reported in Discord and have mentioned or wonder why things don't show in viewport despite the eye icon - so it's not just me. But yeah, thought it minor issue this time due to thinking it was like Bforartists 1 all this time.

**Comment by ReinerBforartists** [comment]
Yeah, but what about the average user? He just uses the eye icon ...

**Comment by Draise14** [comment]
Blender splitting one show/hide operator two viewport visibility toggles was imo really a bad choice, with no real gains... this wouldn't even be a debate if it were just one toggle you could animate, now one is un-animatable, and the other useful one hidden by default. 

Sigh, I guess I'll close for the sake of visual noise. I'll set it up to my own templates and push through customizing this over and over.

**Comment by ReinerBforartists** [comment]
I feel with you !
